src/components/Property/PropertyFlags/index.js

```diff
@@ -1,17 +1,19 @@
 import PropTypes from 'prop-types'
 import { frontEndApiRequest } from '@/utils/frontEndApiClient/requests'
-import Alerts from './Alerts'
-import Tenure from './Tenure'
-import TenureDetail from './TenureDetail'
+import Alerts from '../Alerts'
+import Tenure from '../Tenure'
+import TenureDetail from '../TenureDetail'
 import { useEffect, useState } from 'react'
 import Spinner from '@/components/Spinner'
 import ErrorMessage from '@/components/Errors/ErrorMessage'
+import PropertyBoilerHouseDetails from '../PropertyBoilerHouseDetails'
 
-const TenureDetails = ({
```

Comment:
It might just be worth surfacing this [good] renaming decision a bit more widely, as several teams have worked with this in the past and might be confused

Reply:
Who do you think this would be relevant for? Team houseboat?